### PR TITLE
refactor martenbuild.cs a little

### PR DESCRIPTION
**Pull request by adamralph**

### CODE REVIEW

**Comment by adamralph** [comment]
It seems the CI gods are on my side today. Both builds green first time! :feelsgood:

File: martenbuild.cs [hunk 1]
```diff
@@ -8,46 +8,33 @@ namespace martenbuild
...
             Target("connection", () =>
             {
-                File.WriteAllText("src/Marten.Testing/connection.txt", connection);
+                File.WriteAllText("src/Marten.Testing/connection.txt", Environment.GetEnvironmentVariable("connection"));
             });
 
             Target("install", () =>
             {
-                if (pid == PlatformID.Unix || pid == PlatformID.MacOSX)
+                if (platformID == PlatformID.Unix || platformID == PlatformID.MacOSX)
```
Comment (mysticmind) [inline martenbuild.cs]: Another refactoring area I am observing is checking for the platform, have a helper method as below
```csharp
private static bool IsWindowsPlatform()
{
    var platformID = Environment.OSVersion.Platform;
    
    if (platformID == PlatformID.Unix || platformID == PlatformID.MacOSX)
    {
	return false;
    }

    return true;
}

// change the appropriate code as below to simplify it
IsWindowsPlatform ? Run("cmd.exe", "/c npm run test") : Run("npm", "run test");
```

**Comment by mysticmind** [comment]
@adamralph Thanks for the refactoring changes!

File: martenbuild.cs (hunk 1 above)
Comment (mysticmind) [reply martenbuild.cs]: @adamralph I will take care of applying the above.

Comment (adamralph) [reply martenbuild.cs]: @mysticmind given that the choice between `Run("npm", "...");` and `Run("cmd.exe", "/c npm ...");` is made in two places (`install` and `mocha` targets) I would consider creating a `RunNpm(string args)` method to wrap that decision.

**Pull request merged**